cmd/dlt/externalauthproviders/cmd.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package externalauthproviders
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/interactive/confirm"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+var Cmd = &cobra.Command{
+	Use:     "external-auth-provider",
+	Aliases: []string{"externalauthproviders", "externalauthprovider", "external-auth-providers"},
+	Short:   "Delete cluster external auth",
+	Long:    "Delete a specific external authentication provider for a cluster.",
+	Example: `  # Delete an external authentication provider named exauth-1
+  rosa delete external-auth-provider exauth-1  --cluster=mycluster`,
+	Run:    run,
+	Hidden: true,
+	Args: func(_ *cobra.Command, argv []string) error {
+		if len(argv) != 1 {
+			return fmt.Errorf(
+				"expected exactly one command line parameter containing the name of the external authentication provider",
```

Comment:
Consistency with error message. This one starts with lower case "expected". In run function, messages start with upper case.